app/javascript/layouts/student_course/components/StudentCourse__Header.res

```diff
@@ -58,8 +58,15 @@ let renderCourseSelector = (currentCourseId, courses, coverImage, additionalLink
     courses,
   )
   let otherCourses = Js.Array.filter(c => CourseInfo.id(c) != currentCourseId, courses)
+  let imageWrapperClasses =
+    "relative " ++
+    switch coverImage {
+    | Some(_) => "pb-1/2 md:pb-1/4 2xl:pb-1/5"
+    | None => "pb-1/4 sm:1/5 md:pb-1/6 xl:pb-1/12"
+    }
```

Comment:
@developedBySJ Can we move `imageWrapperClasses` out of `renderCourseSelector`? So that we will just care about the render logic over here.

Reply:
@irajsuhail I will update the code.